build-support/bin/generate_builtin_lockfiles.py

```diff
@@ -226,9 +226,6 @@ def req_file(_tool: PythonTool) -> str:
             "--backend-packages=pants.backend.python",
             "--python-pip-version=23.0.1",
             f"--python-interpreter-constraints=['{default_python_interpreter_constraints}']",
-            # `generate_all_lockfiles.sh` will have overridden this option to solve the chicken
```

Comment:
There is no more chicken/egg problem, since this script doesn't generate the python-default lockfile, and so doesn't care about changes to it.